src/plugin.jl

```diff
@@ -346,6 +346,18 @@ you should implement this function and return `true`.
 """
 needs_username(::Plugin) = false
 
+"""
+    pkg_name_from_pkg_dir(pkg_dir::AbstractString)
+
+Return package name of package at `pkg_dir`, i.e., `basename(pkg_dir)` excluding any
+`.jl` suffix, if present. For example, `foo/bar/Whee.jl` and `foo/bar/Whee` both
+return `Whee`.
+"""
+function pkg_name_from_pkg_dir(pkg_dir::AbstractString)
```

Comment:
maybe just name this `pkg_name`, since it's called like `pkg_name(pkg_dir)`? https://github.com/invenia/BlueStyle#function-naming

```suggestion
function pkg_name(pkg_dir::AbstractString)
```

Reply:
done!